analyzer/analyzer.go

```diff
@@ -233,11 +233,11 @@ func (n *perfSprint) run(pass *analysis.Pass) (interface{}, error) {
 				neededPackages[fname] = make(map[string]bool)
 			}
 			neededPackages[fname]["strconv"] = true
-			d = &analysis.Diagnostic{
-				Pos:     call.Pos(),
-				End:     call.End(),
-				Message: fn + " can be replaced with faster strconv.FormatBool",
-				SuggestedFixes: []analysis.SuggestedFix{
+			d = newAnalysisDiagnostic(
+				"", // TODO: precise checker
```

Comment:
Thanks for working on this

You help me to think further about my code suggestion here

https://github.com/catenacyber/perfsprint/pull/32#issuecomment-2487668199

Yes, these linters not only needs a name, reported as a prefix in the error message, but also they need an option to deactivate each option

Reply:
I guess a way to link the checker name with a CLI option to enable/disable it would be nice indeed.